src/codeu/chat/server/Server.java

```diff
@@ -71,6 +73,14 @@ public Server(final Uuid id, final Secret secret, final Relay relay) {
     this.controller = new Controller(id, model);
     this.relay = relay;
 
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
+      @Override
+      public void onMessage(InputStream in, OutputStream out) throws IOException {
+        Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
+        Time.SERIALIZER.write(out, info.startTime);
```

Comment:
You'll need to change this line after you move the info object - it should look like it does on page 7